src/App.js

```diff
@@ -133,7 +142,6 @@ export default function App() {
                       {wasSearched && (
                         <PagingInfo view={CustomPagingInfoView} />
                       )}
-                      {wasSearched && <ResultsPerPage />}
```

Comment:
If we are dropping `ResultsPerPage` we should remove the import on line 14

Reply:
Alright done